packages/legend-graph/src/models/protocols/pure/v1/transformation/pureProtocol/serializationHelpers/V1_RuntimeSerializationHelper.ts

```diff
@@ -119,6 +125,9 @@ export const V1_packageableRuntimeModelSchema = createModelSchema(
     ),
     name: primitive(),
     package: primitive(),
-    runtimeValue: usingModelSchema(V1_engineRuntimeModelSchema),
+    runtimeValue: custom(
```

Comment:
same as above

Reply:
Tried using Object(V1_EngineRuntime).  It was giving me an error: propDef.deserializer is not a function